backend-implicaction/src/test/java/com/dynonuggets/refonteimplicaction/controller/JobsPostingControllerTest.java

```diff
@@ -154,6 +155,44 @@ void archiveJobShouldChangeStatus() throws Exception {
         verify(jobPostingService, times(1)).toggleArchiveJobPosting(anyLong());
     }
 
+    @Test
+    @WithMockUser
+    void archiveJobShouldChangeListStatus() throws Exception {
+        // given
+        JobPostingDto job = JobPostingDto.builder()
+                .id(1L)
+                .archive(false)
+                .build();
+
+        List<Long> givenDto = Collections.singletonList(job.getId());
+
+        List<JobPostingDto> expectedDto = Collections.singletonList(
+                JobPostingDto.builder()
+                        .id(1L)
+                        .archive(true)
+                        .build()
+        );
+
+        String json = gson.toJson(givenDto);
+
+        given(jobPostingService.toggleArchiveAll(Collections.singletonList(anyLong()))).willReturn(expectedDto);
+
+        // when
+        final ResultActions resultActions = mvc.perform(patch(JOBS_BASE_URI + ARCHIVE_JOBS_URI).contentType(APPLICATION_JSON).content(json));
+
+        // then
+        resultActions.andDo(print())
+                .andExpect(status().isOk());
+
+        for (int i = 0; i < givenDto.size(); i++) {
+            final String contentPath = String.format("$[%d]", i);
+            resultActions
+                    .andExpect(jsonPath(contentPath + ".id", is(job.getId())))
```

Comment:
```suggestion
                    .andExpect(jsonPath(contentPath + ".id", is(job.getId().intValue())))
```
Le teste devrait échouer je pense parce qu'il essaye de comparer un int (`$[x].id`) à un long (`job.getId()`)